### ISSUE TITLE
Windows support / testing

### ISSUE THREAD

**Issue by ghost** [johnboiles/JBKinectHacks#2]
you need to make this available for pc kinect it is partly made by microsoft anyway.

**Comment by johnboiles** [comment]
I totally agree. I don't have a windows machine though. Feel free to work on windows support! I think most of this should work for windows without much work. You'll probably have to change the networking calls in UDPBackend at least

**Comment by ghost** [comment]
lol i don't know much about programming but hopefully someone else will make a PC version and i thought you just didn't want a PC version

**Comment by ghost** [comment]
I'm suprized you don't consider using bootcamp for mac. it should allow you to run windows to do dev.
